content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,130 +1,60 @@
-`git checkout` and `dvc checkout` to perform full or partial
-<abbr>workspace</abbr> restorations.
-
-![](/img/versioning.png) _Code and data checkout_
-
-> Note that `dvc install` enables auto-checkouts of data after `git checkout`.
-
-A full checkout brings the whole <abbr>project</abbr> back to a previous version
-— code, dataset and model files all match each other:
-
-```dvc
-$ git checkout v1.0
-$ dvc checkout
-M       images
-M       model.pkl
-```
-
-However, we can checkout certain parts only, for example if we want to keep the
-latest source code and model versions, but rewind to the previous version of the
-dataset:
-
-```dvc
-$ git checkout v1.0 images.dvc
-$ dvc checkout images.dvc
-M       images
-```
-
-DVC [optimizes](/doc/user-guide/large-dataset-optimization) this operation by
-avoiding copying files each time, so checking out data is quick even if you are
-versioning large data files.
+# Versioning Data and Models
+
+Data science teams today face data management questions around versioning
+datasets, data artifacts, and machine learning models. How do we keep track of
+changes in data, code, and ML models together? What's the best way to organize
+and store multiple versions of data and model files? How can data lifecycles be
+defined and enforced?
+
+![](/img/data-ver-complex.png) _Exponential complexity of data science projects_
+
+Data Version Control (DVC) can address these issues by helping you organize your
+data and models effectively and capture their versions in
+[Git commits](<(https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository)>),
+while storing the actual data contents outside ot Git. This way there's a single
+project history for all of its assets and artifacts, and at the same time the
+data stays with you.
+
+![](/img/project-versions.png) _DVC matches the right versions of code and
+data._
+
+DVC enables data _versioning through codification_. This means describing which
+data, ML artifacts, etc. should be in the environment at any given time. This is
+achieved with special [metafiles](/doc/user-guide/dvc-files-and-directories)
+that can be put in Git — but we won't much deeper into how it works here, since
+we have many [guides](/doc/user-guide) and [references](/doc/command-reference)
+for that. To learn how it looks and feels, try our
+[versioning tutorial](/doc/use-cases/versioning-data-and-model-files/tutorial)
+👩‍💻.
+
+DVC's approach brings the following benefits:
+
+- **Tidy project**: Work with a natural file structure. No need for ad hoc
+  naming conventions like `data/20190922/labels_v7_final`.
+- **Reproducibility**: Restore any project version and find the corresponding
+  data instantly. Identify past research inputs to understand its results, or
+  for debugging.
+- **Easy storage**: All versions of data are
+  [cached](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory)
```

Comment:
Easy storage - not clear what is it about. What is cache (first reference from this page)? You can remove Easy storage from the list.

Reply:
True. We used to have more info on storage and cache before. Removed!

Reply:
But... I'm starting to think we should cover cache/storage a bit more as part of the story at least, as it's in the "problems" presented initially:

> best way to organize and **store** multiple versions of data and model files?

OR we could just no talk about storage anywhere at all in this doc, leaving that for **Sharing Data and Model Files** (needs rewrite too).

Reply:
there are a few things to storage story that should be ideally covered in the DVC benefits:

1. data stays with you - runs on top of your cloud, on-prem, etc. 
2. efficient data management - local cache that utilizes links, shared cache scenario to share data efficiently

`easy` in this case doesn't provide any meaningful information (what is Hard storage? Easy from what perspective?)

also, no need to emphasize on all versions here - that's what all the previous text is about to my mind

Reply:
1. "data stays with you" is in the story as:

> you can have a single, immutable history for all of your assets and artifacts, and at the same time, the **data stays with you**.

Separately, the **Collaboration** benefit mentions that data can be shared/reused via **on-prem or cloud storage** (which currently is the only mention of "storage" in the doc).

Working on 2. but we may end up with too many bullets again ⌛

Reply:
> 2. efficient data management - local cache... shared cache...

OK I added this benefit but we're back to square one here. The cache mention is kind of out of context, maybe unnecessary. I think it's probably better to revisit the next use case to focus more on storage (both local cache and remotes).

Reply:
cache should not be considered as DVC cache ... there is a general notion of caching that most people would understand. E.g. if you say that `DVC gives you a way to granularly cache data on your local or training machine` most people should understand this to my mind.

Reply:
OK I reworked the doc per the comments above and some private feedback. PTAL